docs/content/preview/migrate/yb-voyager/yb-voyager-cli.md

```diff
@@ -0,0 +1,310 @@
+---
+title: yb-voyager CLI
+linkTitle: yb-voyager CLI
+description: YugabyteDB Voyager CLI and SSL connectivity.
+beta: /preview/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+menu:
+  preview:
+    identifier: yb-voyager-cli
+    parent: yb-voyager
+    weight: 105
+isTocNested: true
+showAsideToc: true
+---
+
+yb-voyager is a command line executable for migrating databases from PostgreSQL, Oracle, and MySQL to a YugabyteDB database.
+
+## Syntax
+
+```sh
+yb_voyager [ <migration-step>... ] [ <arguments> ... ]
+```
+
+- *migration-step*: See [Migration steps](#migration-steps)
+- *arguments*: See [Arguments](#arguments)
+
+### Command line help
+
+To display the available online help, run:
+
+```sh
+yb_voyager --help
+```
+
+To display the available online help for any migration step, run:
+
+```sh
+yb_voyager [ <migration-step>... ] --help
+```
+
+### Migration steps
```

Comment:
I'd bump up the heading level of this and Arguments; if you look at other CLI pages, commands and arguments aren't placed under syntax heading. Also, rename this simply "Commands" or somesuch - for the reference this isn't really the steps